CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# main
```

Comment:
nit : we can add it as `10.4.0-rc.1`. not required though. we can change that later :D

Reply:
It was just changed from `10.4.0-rc.1` to `main`. due to the comment in https://github.com/mapbox/mapbox-maps-android/pull/1183#discussion_r814007886